docker/composer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM composer/composer
```

Comment:
If you want, i can remove the container. AFAIK i can just add it to my local `docker-compose.override.yml`